mobile/examples/image_classifications/android/app/src/main/java/ai/onnxruntime/example/imageclassifier/ImageUtil.kt

```diff
@@ -0,0 +1,174 @@
+package ai.onnxruntime.example.imageclassifier
```

Comment:
Need add license title to this file, most of it is from AOSP project
```suggestion
/*
 * Copyright 2020 The Android Open Source Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package ai.onnxruntime.example.imageclassifier
```